UKHomeOffice/cop-ui
# COP-9243 Update you do not have team page

Ticket: https://support.cop.homeoffice.gov.uk/browse/COP-9243

In this PR
- Update `You do not have team` page
- New Figma design https://www.figma.com/file/HvHakMNbQ9Z17Arw8flhuy/COP?node-id=3163%3A8990


![image](https://user-images.githubusercontent.com/30794075/142472550-41cad098-e0c8-48c7-b032-2586caf9c9e6.png)


# Testing instructions

- Remove all groups form keycloak
- Sign in to COP
- You should see new page layout
